src/sandstorm/backup.c++

```diff
@@ -106,9 +106,6 @@ void BackupMain::bind(kj::StringPtr src, kj::StringPtr dst, unsigned long flags)
 }
 
 bool BackupMain::run(kj::StringPtr grainDir) {
-  KJ_ASSERT(geteuid() != 0, "can't run as user ID zero (root)");
```

Comment:
I don't see how these assert() changes in backup.c++ are relevant. Does that mean on the privileged sandbox, downloading grain backups theoretically doesn't work right now? Let me go check that.